commercial/app/model/commercial/CapiImages.scala

```diff
@@ -0,0 +1,58 @@
+package model.commercial
+
+import views.support.ImgSrc
+import cards.Standard
+import layout.{FaciaWidths, ItemClasses}
+import model.ImageMedia
+import play.api.libs.json.{Json, Writes}
+
+object CapiImages {
+
+  // Puts together image source info using data from cAPI.
+  def buildImageData(imageData: Option[ImageMedia]): ImageInfo = {
+
+    val fallbackImageUrl = imageData flatMap ImgSrc.getFallbackUrl
+    val cardType = Standard
+
+    val breakpointWidths = FaciaWidths.mediaFromItemClasses(ItemClasses(
+      mobile = Standard,
```

Comment:
Hmm, looking at that you've made me wonder whether I hardcoded something while I was building something else and then forgot about it. I'll double-check.